pkg/server/ca/journal.go

```diff
@@ -188,14 +188,12 @@ func migrateJSONFile(from, to string) (bool, error) {
 		if len(chain) == 0 {
 			return nil, nil
 		}
-		// The chain is in one of three states:
+		// The chain is in one of two states:
```

Comment:
Do we need any update on the test side?

Reply:
I mean journal_test.go

Reply:
Sure! The production code isn't impacted by the change but I can see how the unit-test case could be confused. Removed the test case that didn't have a bundle.